src/StatsdClient/Serializer/MetricSerializer.cs

```diff
@@ -71,7 +71,7 @@ private void AppendDouble(StringBuilder builder, double v)
             var intValue = (int)v;
             var provider = CultureInfo.InvariantCulture;
 
-#if HAS_SPAN
+#if NETCOREAPP3_1_OR_GREATER
```

Comment:
Span is available in 2.1 🙂 

```suggestion
#if NETCOREAPP2_1_OR_GREATER
```